doc/dev.md

```diff
@@ -191,10 +166,26 @@ organized into different subdirectories:
 5. `x86`: x86-64 machine-code CFGs (via `macaw-x86-syntax`). Each of these test
    cases has the file exension `*.x64.cbl`.
 
-These test-cases may override the default command-line options using
-specially-formatted comments. For LLVM bitcode programs, add a line beginning
-with `// flags: ` to the corresponding C program. For S-expression programs, add
-a line beginning with `; flags: `.
+### Lua API
+
+The test harness provides the following Lua API, which enriches that of Oughta
+with GREASE-specific functionality. The API is presented with Haskell-style
+type signatures:
+
+- `prog :: String`: The name of the program under test, e.g., `test.llvm.cbl`.
+- `go :: String -> ()`: Run GREASE with the given string as the name of the
+  program. Frequently, `prog` is passed as the argument, e.g., `go(prog)`.
+- `flags :: [String] -> ()`: Append the given flags to the arguments passed to
+  GREASE when `go` is invoked.
```

Comment:
It's a bit weird that we are describing Lua code with Haskell type signatures here. There's nothing _wrong_ with doing so per se, but the mixture of the two languages is a bit confusing. I wonder if we should instead present these like Lua API docs, i.e., something like:

> ```lua
> go(prog_name)
> ```
>
> Run GREASE with the given string as the name of the program [...]

(Inspired by the [`lua-cjson` docs](https://kyne.au/~mark/software/lua-cjson-manual.html))

Reply:
Sure, will do!